package.json

```diff
@@ -15,7 +15,7 @@
     "export": "vuepress export src"
   },
   "devDependencies": {
-    "@vuepress/plugin-google-analytics": "^1.8.2",
+    "vuepress-plugin-google-tag-manager": "*",
```

Comment:
Should we be explicit with the version here?

Reply:
yea, I can put it here, however it is bit experimental plugin with version 0.0.3 or something like that